spec.bs

```diff
@@ -634,7 +634,7 @@ The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps are
     "[=join-ad-interest-group=]" [=policy-controlled feature=], then [=exception/throw=] a
     "{{NotAllowedError}}" {{DOMException}}.
 
-    Note: Both joining and leaving interest groups use the "join-ad-interest-group" feature.
+        Note: Both joining and leaving interest groups use the "join-ad-interest-group" feature.

```

Comment:
is this change needed?

Reply:
I have no idea as to what happened here. Probably confused with my own note somehow? Removed.